virtcontainers/api.go

```diff
@@ -653,3 +653,23 @@ func StatsContainer(sandboxID, containerID string) (ContainerStats, error) {
 
 	return s.StatsContainer(containerID)
 }
+
+// AddNetwork is the virtcontainers entry point to add a new nic.
+func AddNetwork(sandboxID, ifName string) error {
+	if sandboxID == "" {
+		return errNeedSandboxID
+	}
+
+	lockFile, err := rLockSandbox(sandboxID)
```

Comment:
This should be a RW lock as I expect this function to store the newly added interface at some point.